extensions/window/init.lua

```diff
@@ -25,30 +23,131 @@ local atan,cos,abs = math.atan,math.cos,math.abs
 --- hs.window.animationDuration = 3 -- if you have time on your hands
 window.animationDuration = 0.2
 
---- hs.window.allWindows() -> win[]
---- Function
+--- hs.window.allWindows() -> list of hs.window objects
+--- Constructor
```

Comment:
I don't think this is a constructor - they should typically be returning an object that has a metatable.